src/librustc/diagnostics.rs

```diff
@@ -947,6 +945,8 @@ position that needs that trait. For example, when the following code is
 compiled:
 
 ```compile_fail
+#![feature(on_unimplemented)]
+
```

Comment:
Why did these examples start requiring `#![feature(on_unimplemented)]` but still not  get error code annotations?